pkg/querier/frontend/frontend.go

```diff
@@ -210,7 +210,7 @@ func (f *Frontend) RoundTripGRPC(ctx context.Context, req *ProcessRequest) (*Pro
 	tracer, span := opentracing.GlobalTracer(), opentracing.SpanFromContext(ctx)
 	if tracer != nil && span != nil {
 		carrier := (*httpgrpcHeadersCarrier)(req.HttpRequest)
-		tracer.Inject(span.Context(), opentracing.HTTPHeaders, carrier)
+		_ = tracer.Inject(span.Context(), opentracing.HTTPHeaders, carrier)
```

Comment:
@bboreham Should we add `tracer.Inject` to the exclude list?

Reply:
sure, whatever.

Reply:
@zendern May you do it, please?

Reply:
Fixed in ccca019